kubernetes/helm/pinot/templates/broker/service-headless.yaml

```diff
@@ -22,17 +22,12 @@ kind: Service
 metadata:
   name: {{ template "pinot.broker.headless" . }}
   labels:
-    app: {{ include "pinot.name" . }}
-    chart: {{ include "pinot.chart" . }}
-    component: {{ .Values.broker.name }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    {{- include "pinot.brokerLabels" . | nindent 4 }}
 spec:
-  clusterIP: None
+  clusterIP: {{ .Values.broker.service.clusterIP }}
```

Comment:
I think this is the convention for headless services: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-hostname-and-subdomain-fields

Reply:
Copy-paste err, Updated